app/views/recruitment_cycles/_allocations.html.erb

```diff
@@ -4,13 +4,10 @@
                       provider_recruitment_cycle_allocations_path(provider.provider_code, year) %>
   </h2>
 
-  <p class="govuk-body">
-    <%= t("allocations_for_pe.#{Allocation.journey_mode}_state_copy") %>
-  </p>
-
-  <p class="govuk-body">This includes:</p>
+  <p class="govuk-body">Use this section to:</p>
   <ul class="govuk-list govuk-list--bullet">
-    <li>your own courses</li>
-    <li>courses you’re the accredited body for (you must request these on behalf of training partners)</li>
+    <li>request fee-funded PE courses (ones without a salary)</li>
+    <li>check the status of these requests</li>
```

Comment:
This would probably introduce more support tickets so can we remove, please

Reply:
This copy came over from the prototype… I’m guessing this is talking about features that don’t exist on prod?

Reply:
Correct